include/tvm/tir/schedule/schedule.h

```diff
@@ -299,6 +302,20 @@ class Schedule : public runtime::ObjectRef {
    */
   TVM_DLL static Schedule Concrete(IRModule mod, int debug_mode,
                                    ScheduleErrorRenderLevel error_render_level);
+  /*!
+   * \brief Construct a traced concrete TensorIR schedule from an IRModule
+   * \param mod The IRModule to be scheduled
+   * \param debug_mode Do extra correctness checking after the class creation
+   * and each time after calling the Replace method.
+   * \param error_render_level The level of error rendering
+   * \return The concrete schedule created
+   * \sa ScheduleDebugMask
+   * \note The checks performed include:
+   * 1) VerifySRefTree
+   * 2) VerifyCachedFlags
+   */
+  TVM_DLL static Schedule Traced(IRModule mod, int debug_mode,
+                                 ScheduleErrorRenderLevel error_render_level);
```

Comment:
Maybe `TracedSchedule` is more precise. So does `ConcreteSchedule`.

Reply:
Yeah Cody and I just had a debate on the naming of this factory method, and finally we agreed to move to `ConcreteSchedule` and `TracedSchedule`, which makes it crystal clear although sounds repetitive :-)